draft-ietf-jsonpath-base.md

```diff
@@ -231,6 +224,8 @@ Normalized Path:
 : A simple form of JSONPath expression that identifies a node by
   providing a query that results in exactly that node.  Similar
   to, but syntactically different from, a JSON Pointer {{-pointer}}.
+  Note: Normalized Paths are not canonical: two distinct Normalized Paths
```

Comment:
This cries for an example.
It probably would be useful to have a place where normalized paths are discussed that is outside the definitions.

Reply:
Ok, I'll bite.